src/containers/forgotPassword/index.tsx

```diff
@@ -12,16 +12,21 @@ import { enterEmailRules } from '../../utils/formRules';
 const ForgotPassword: React.FC = () => {
   const { windowType } = useWindowDimensions();
 
+  const [submittedEmail, setSubmittedEmail] = useState(false);
+  const [email, setEmail] = useState('');
```

Comment:
⛏️ Could maybe just have `email` and check if it's null or empty string to check whether or not they've submitted the email.